src/clients/client.ts

```diff
@@ -76,21 +80,24 @@ class Client {
    * @memberof MeiliSearch
    * @method getRawIndex
    * @param {string} indexUid The index UID
-   * @returns {Promise<IndexResponse>} Promise returning index information
+   * @returns {Promise<IndexObject>} Promise returning index information
```

Comment:
It seems a kind of https://en.wikipedia.org/wiki/Hungarian_notation not sure if this is the best choice, why not just `Index` are you using it somewhere else??